Fix "make dist" in `console-client'.  In particular, make it so that
`kdioctlServer.c' isn't built and distributed by "make dist".

diff --git a/console-client/Makefile b/console-client/Makefile
index bf1335b..f861546 100644
--- a/console-client/Makefile
+++ b/console-client/Makefile
@@ -35,7 +35,8 @@ SRCS = $(CONSOLE_SRCS) \
   $(GENERIC_SPEAKER_SO_SRCS) $(CURRENT_VCS_SO_SRCS) $(NCURSESW_SO_SRCS)
 LCLHDRS = timer.h driver.h display.h input.h bell.h \
 	  unicode.h bdf.h mach-inputdev.h \
-	  vga-dynafont.h vga-dynacolor.h vga-hw.h vga.h
+	  vga-dynafont.h vga-dynacolor.h vga-hw.h vga-support.h \
+	  trans.h
 
 OBJS = $(SRCS:.c=.o)
 HURDLIBS = cons threads ports netfs fshelp iohelp ihash shouldbeinlibc

diff --git a/console-client/Makefile b/console-client/Makefile
index bf1335b..58575c1 100644
--- a/console-client/Makefile
+++ b/console-client/Makefile
@@ -22,8 +22,7 @@ makemode := utilities
 targets = console
 CONSOLE_SRCS = console.c timer.c driver.c trans.c
 VGA_SO_SRCS = bdf.c vga-dynafont.c vga-dynacolor.c vga-support.c vga.c
-#PC_KBD_SO_SRCS = pc-kbd.c kdioctlServer.o kbd-repeat.c
-PC_KBD_SO_SRCS = pc-kbd.c kdioctlServer.c kbd-repeat.c
+PC_KBD_SO_SRCS = pc-kbd.c kbd-repeat.c
 PC_MOUSE_SO_SRCS = pc-mouse.c
 GENERIC_SPEAKER_SO_SRCS = generic-speaker.c
 CURRENT_VCS_SO_SRCS = current-vcs.c
@@ -37,7 +36,7 @@ LCLHDRS = timer.h driver.h display.h input.h bell.h \
 	  unicode.h bdf.h mach-inputdev.h \
 	  vga-dynafont.h vga-dynacolor.h vga-hw.h vga.h
 
-OBJS = $(SRCS:.c=.o)
+OBJS = $(SRCS:.c=.o) kdioctlServer.o
 HURDLIBS = cons threads ports netfs fshelp iohelp ihash shouldbeinlibc
 LDLIBS = -ldl
 module-dir = $(libdir)/hurd/console
@@ -60,7 +59,8 @@ modules = vga pc_kbd generic_speaker pc_mouse current_vcs
 
 vga-CPPFLAGS = -DDEFAULT_VGA_FONT_DIR=\"${datadir}/hurd/\"
 vga.so.$(hurd-version): $(patsubst %.c,%_pic.o,$(VGA_SO_SRCS))
-pc_kbd.so.$(hurd-version): $(patsubst %.c,%_pic.o,$(PC_KBD_SO_SRCS))
+pc_kbd.so.$(hurd-version): $(patsubst %.c,%_pic.o,$(PC_KBD_SO_SRCS))	\
+	kdioctlServer_pic.o
 pc_mouse.so.$(hurd-version): $(patsubst %.c,%_pic.o,$(PC_MOUSE_SO_SRCS))
 generic_speaker.so.$(hurd-version): $(patsubst %.c,%_pic.o,$(GENERIC_SPEAKER_SO_SRCS))
 current_vcs.so.$(hurd-version): $(patsubst %.c,%_pic.o,$(CURRENT_VCS_SO_SRCS))
